src/style.css

```diff
@@ -1,3 +1,92 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  height: 60px;
+  background-color: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0;
+  position: relative;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
+}
+
+.is-active {
+  color: black;
+  text-decoration: none;
+  position: relative;
+  display: inline-block;
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: var(--main-color);
+  position: absolute;
+  bottom: 0;
+  border-radius: 8px;
+}
+
+.nav__link {
+  display: inline-block;
+  height: 60px;
+  line-height: 60px;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 0 0 0 20px;
+  text-align: center;
+}
+
+.nav__link:hover {
+  height: 60px;
+  line-height: 60px;
+  text-decoration: underline;
+  text-decoration-color: var(--main-color);
+  text-decoration-thickness: 2px;
+}
+
+.nav__item:nth-child(4) .nav__link:hover {
+  color: var(--main-color);
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  padding: 10px 0 10px 50px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  list-style: none;
+  padding: 0 50px 0 0;
+}
+
+.logo {
+  display: inline-block;
+}
+
+.nav__list .nav__item:nth-child(1) .nav__link {
+  padding: 0;
+  color: var(--main-color);
+}
```

Comment:
```suggestion
.nav__list .nav__item:first-child .nav__link {
  margin-right: 0;
}
```

Reply:
add color to the .is-active

Reply:
if margin-right:0  for the first child there isn`t space between Apple and Samsung